review/ha_basic.toml

```diff
@@ -0,0 +1,44 @@
+[server]
+hostname = "wso2is.example.com"
+proxy_port = "443"
+local_ip = "192.168.1.1"
```

Comment:
can not we derive this?

Reply:
This may not be possible if there are multiple network interfaces for the machine. That's why we decided to add this specifically.